src/pointer-analysis/value_set.cpp

```diff
@@ -22,6 +22,8 @@ Author: Daniel Kroening, kroening@kroening.com
 
 #include <langapi/language_util.h>
 #include <util/range.h>
+#include <util/format_expr.h>
+#include <util/format_type.h>
```

Comment:
But then `#include <langapi/language_util.h>` better be removed as well (and hopefully also the corresponding entry in `module_dependencies.txt`).